### DISCUSSION TITLE
[Jaeger Query] Read traces from elasticsearch that are exported by Open Telemetry

### DISCUSSION THREAD

**Discussion by PandeoF1** [jaegertracing/jaeger#5344] [Q&A] [answered]
Hi, it is possible to read data that are exported by OTEL and not Jaeger Collector in Elasticsearch.
Because actually i have: otel collector -> Jaeger collector -> elastic -> jaeger query.
I have set OTEL to export data in elastic directly but Jaeger query don't find my traces. (I can saw them in elastic)
So it is possible to have only OTEL -> Elastic -> Jaeger Query ?

**Comment by yurishkuro** [comment, marked as answer]
Not at the moment. Once jaeger-v2 is available you could use it instead of OTEL-collector.

**Comment by PandeoF1** [comment]
Thanks for you response, i can't remove my OTEL-collector because it's used for a lot of thing in my architecture but i will look that in the future when it will be available.
So i will keep the jaeger collector !
